src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/MultiPipelineGraph.tsx

```diff
@@ -18,27 +18,35 @@ export const MultiPipelineGraph = () => {
       });
     }
   }, [runs, poll]);
+
+  const groupedRuns: Record<string, RunInfo[]> = runs.reduce(
+    (acc: Record<string, RunInfo[]>, run) => {
+      const date = new Date(run.timestamp).toLocaleDateString("en-US", {
```

Comment:
do we want this based on the users locale?

Reply:
https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/647